modules/ossm-operatorhub-install.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-operatorhub-install_{context}"]
+= Installing the Operators from OperatorHub
+
+The {ProductShortName} installation process uses the link:https://operatorhub.io/[OperatorHub] to install the `ServiceMeshControlPlane` custom resource definition within the `openshift-operators` project. The {ProductName} defines and monitors the `ServiceMeshControlPlane` related to the deployment, update, and deletion of the control plane.
+
+Starting with {ProductName} {ProductVersion}, you must install the Elasticsearch Operator, the Jaeger Operator, and the Kiali Operator before the {ProductName} Operator can install the control plane.
+
+[id="ossm-operator-install-elasticsearch_{context}"]
+== Installing the Elasticsearch Operator
+You must install the Eliasticsearch Operator for the {ProductName} Operator to install the control plane.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Prerequisites
+* Access to the {product-title} web console.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalog* -> *OperatorHub*.
+
+. Type *Elasticsearch* into the filter box to locate the Elasticsearch Operator.
+
+. Click the *Elasticsearch Operator* to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and is made available to all projects in the cluster.
+
+. Select the *preview* Update Channel.
+
+. Select the *Automatic* Approval Strategy.
++
+[NOTE]
+====
+The Manual approval strategy requires a user with appropriate credentials to approve the Operator install and subscription process.
+====
+
+. Click *Subscribe*.
+
+. The *Subscription Overview* page displays the Elasticsearch Operator's install process.
+
+
+[id="ossm-operator-install-jaeger_{context}"]
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Prerequisites
+* Access to the {product-title} web console.
+* The Elasticsearch Operator must be installed.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Navigate to *Catalog* -> *OperatorHub*.
+
+. Type *Jaeger* into the filter box to locate the Jaeger Operator.
+
+. Click the *Jaeger Operator* provided by Red Hat to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and is made available to all projects in the cluster.
```

Comment:
Possibly: "This installs the Operator in the default `openshift-operators` project and _makes the Operator_ available to all projects in the cluster."

Reply:
Fixed.